### PR TITLE
Modularize

### PR DESCRIPTION
**Pull request by calvinmetcalf**
also add standard styling and other 3 curves

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
ping @crypto-browserify/owners

**Comment by jprichardson** [comment]
ACK

Is https://github.com/crypto-browserify/browserify-sign/commit/d2a3f77b244ce8e967bd2b3067cdfaee670dfecf the start of our adherence to https://github.com/feross/standard?

File: curves.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+exports['1.3.132.0.10'] = 'secp256k1'
+
+exports['1.3.132.0.33'] = 'p224'
```
Comment (dcousens) [inline curves.js]: what type of aliases are these?  I haven't seen them before?  :)

**Comment by dcousens** [comment]
@calvinmetcalf reviewing these PRs is really difficult because you mix functional changes, formatting changes and restructuring changes.

Can you please isolate them in future?  I don't mind if they're in the same PR, but please, not the same commits at least.   It already takes a long time when what looks like basic module extraction actually has a few subtle changes that I then means I have to diff everything to verify it...

File: curves.js (hunk 1 above)
Comment (calvinmetcalf) [inline curves.js]: on the left are asn1 oids and on the right are the names that elliptic uses

**Comment by calvinmetcalf** [comment]
yeah my bad though [turning off white space](https://github.com/crypto-browserify/browserify-sign/commit/d2a3f77b244ce8e967bd2b3067cdfaee670dfecf?w=0) makes that one easier.

**Comment by calvinmetcalf** [comment]
ping

**Comment by calvinmetcalf** [comment]
@crypto-browserify/owners I added 3 new commits 

-  48f8881 adds tests to make sure we are checking the padding (and finds we aren't so adds checks to the padding), also fixes a typo
-  3f10450 makes sure all the code is strict to avoid more typos
- 07a8727 adjusts the formatting so that it passes standard

**Comment by jprichardson** [comment]
ACK. I saw https://github.com/crypto-browserify/browserify-sign/commit/3f1045017b95fdb0a0da87fe57b467705edbafdf and then was going to suggest https://github.com/feross/standard but then saw your comments for https://github.com/crypto-browserify/browserify-sign/commit/07a87278e6eb13539ee0a58262df3213e8cea4da so it looks like that's covered :) Cool.

**Comment by dcousens** [comment]
ACK

**Comment by calvinmetcalf** [comment]
published in 3.0.0

**Pull request merged**